cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { configEnvWithTenderlyMainnetFork } from '../../../support/steps/configuration.steps';
+import { skipState } from '../../../support/steps/common';
+import { supply, borrow } from '../../../support/steps/main.steps';
+import constants from '../../../fixtures/constans.json';
+import assets from '../../../fixtures/assets.json';
+
+const testData = {
+  depositETH: {
+    asset: assets.aaveMarket.ETH,
+    amount: 0.5,
+    hasApproval: true,
+  },
+  testCases: {
+    deposit: {
+      asset: assets.aaveMarket.USDT,
+      amount: 50,
+      hasApproval: false,
+    },
+    borrow: [
+      {
+        asset: assets.aaveMarket.USDT,
+        amount: 50,
+        apyType: constants.borrowAPYType.variable,
+        hasApproval: true,
+      },
+    ],
```

Comment:
don't need to us array here, u just have 1 borrow action, and don't need to use second one